### ISSUE TITLE
fileCache.Read err: open /var/folders/dh/0d7rlkkd1tv_klrv21_1_f900000gn/T/fake_useragent_0.2.0.json: too many open files

### ISSUE THREAD

**Issue by supremepot** [batiscuff/tg_booster#1]
fileCache.Read err: open /var/folders/dh/0d7rlkkd1tv_klrv21_1_f900000gn/T/fake_useragent_0.2.0.json: too many open files

how can I solve this?

**Comment by batiscuff** [comment]
tg_booster current support only HTTPS proxies from .txt file. 

UPD: or reduce the number of workers, or increase number of open file descriptors

**Comment by ShadabAlam786** [comment]
please make a code for deploying on heroku.

**Comment by batiscuff** [comment]
> please make a code for deploying on heroku.

I don't think this makes sense anymore.  Since boosting views through a proxy is no longer relevant.
